app/components/forms/wizard/wizard-footer.hbs

```diff
@@ -33,7 +33,7 @@
   </div>
 </div>
 <div class="ui grid">
-  <div class="sixteen wide column right aligned">
+  <div class="sixteen wide column left aligned">
```

Comment:
it shouldn't be left aligned for desktop view.

Reply:
Thanks, should I use if condition for mobile view only?